src/lib/button-toggle/_button-toggle-theme.scss

```diff
@@ -1,13 +1,25 @@
 @import '../core/theming/palette';
 @import '../core/theming/theming';
 
+// Applies a focus style to an md-button-toggle element for each of the supported palettes.
+@mixin _mat-button-toggle-focus-color($theme) {
+  $foreground: map-get($theme, foreground);
+
+  .mat-button-toggle-focus-overlay {
+    background-color: mat-color($foreground, divider);
```

Comment:
Nit: Do you want to add another property as 'dark-overlay' or something? Using a property from "foreground" as a background is a bit confusing out of context.